File: app/src/main/java/com/werockstar/reactiveandroid/adapter/GithubViewHolder.java
package com.werockstar.reactiveandroid.adapter;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.werockstar.reactiveandroid.R;

import butterknife.BindView;
import butterknife.ButterKnife;

public class GithubViewHolder extends RecyclerView.ViewHolder {

    @BindView(R.id.ivAvatar) ImageView ivAvatar;
    @BindView(R.id.tvBlog) TextView tvBlog;
    @BindView(R.id.tvName) TextView tvName;

    public GithubViewHolder(View view) {
        super(view);

        ButterKnife.bind(this, view);
    }
}

File: app/src/main/java/com/werockstar/reactiveandroid/adapter/GithubViewHolder.kt
package com.werockstar.reactiveandroid.adapter

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import com.bumptech.glide.Glide
import com.werockstar.reactiveandroid.R
import com.werockstar.reactiveandroid.model.GithubUserResponse
import kotlinx.android.synthetic.main.github_item_row.view.*

class GithubViewHolder(val view: View) : RecyclerView.ViewHolder(view) {

	fun bind(user: GithubUserResponse) {
		view.tvName.text = user.name
		view.tvBlog.text = user.blog
		Glide.with(view.context).load(user.avatarUrl).error(R.mipmap.ic_launcher_round)
			.into(view.ivAvatar)
	}
}
